Documentation/Metrics/arangodb_vocbase_meta_collection_lock_locked_exclusive.yaml

```diff
@@ -0,0 +1,14 @@
+name: arangodb_vocbase_meta_collection_lock_locked_exclusive
```

Comment:
I don't think we should name a publicly visible metric something like "*vocbase*". "vocbase" is some internal term that doesn't make much sense to end users. The same is true for "meta_collection". 
Why not go with some simpler name such as "arangodb_collection_locks_locked_*"? If that name is already in use, maybe some similar easy alternative?